app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -118,10 +119,16 @@ def redhat_provider
                        :resource_name => "providers")
     end
 
+    api :GET, '/activation_keys/:id/service_levels', N_('List available service levels for an activation key')
```

Comment:
I do agree that the API call is misleading, but according to [this comment](https://bugzilla.redhat.com/show_bug.cgi?id=1144042#c5) the user seems to really feel that there should be an API call to activation keys that list available service levels because the UI references service levels on the activation key's details page.